spec/services/nomis/api_spec.rb

```diff
@@ -261,7 +261,7 @@
     subject { super().fetch_contact_list(params) }
 
     it 'returns an array of contacts' do
-      expect(subject).to have_exactly(27).items
+      expect(subject).to have_exactly(26).items
```

Comment:
slightly worried about this one? Suggests a change in behaviour. Maybe improve test on master to expect actual data so that we can see which contact has been mysteriously removed?

Reply:
This was due to a change in the behaviour of the API and has since been fixed.